.buildkite/scripts/steps/integration_tests_custom_ech_tf.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+source .buildkite/scripts/steps/ess.sh
+source .buildkite/scripts/steps/fleet.sh
+
+# Make sure that all tools are installed
+asdf install
+
+# Override the agent package version using a string with format <major>.<minor>.<patch>
+# There is a time when the snapshot is not built yet, so we cannot use the latest version automatically
+# This file is managed by an automation (mage integration:UpdateAgentPackageVersion) that check if the snapshot is ready.
+OVERRIDE_STACK_VERSION="$(cat .package-version)"
+OVERRIDE_STACK_VERSION=${OVERRIDE_STACK_VERSION}"-SNAPSHOT"
+
+echo "~~~ Building test binaries"
+mage build:testBinaries
+
+# Remove deployment on exit
+trap 'ess_down' EXIT
+
+preinstall_fleet_packages
+
+# Run integration tests
+echo "~~~ Running sudo ECH integration tests"
+./buildkite/scripts/buildkite-integration-tests.sh ech true
```

Comment:
let's keep new shell scripts at a minimum please 🙏  IIUC you can do what you need to do, just by introducing a new env var here and make this script [here](https://github.com/elastic/elastic-agent/blob/14ebe7e89eef53240f215a166e90eb5ac6486403/.buildkite/scripts/steps/integration_tests_tf.sh#L35) always provision an ECH stack e.g. `ESS_CREATE`, @v1v such env var makes sense to me, wdyt? 🙂